<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>677</width>
    <height>667</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>440</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>PyCirkuit - by Orestes Mas</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources/resources.qrc">
    <normaloff>:/icons/AppIcon</normaloff>:/icons/AppIcon</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="font">
       <font>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_source">
       <attribute name="icon">
        <iconset resource="../resources/resources.qrc">
         <normaloff>:/icons/Text</normaloff>:/icons/Text</iconset>
       </attribute>
       <attribute name="title">
        <string comment="Tab title">Source Editor</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="pycktTextEditor" name="sourceText">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_log">
       <attribute name="icon">
        <iconset resource="../resources/resources.qrc">
         <normaloff>:/icons/Output</normaloff>:/icons/Output</iconset>
       </attribute>
       <attribute name="title">
        <string comment="Tab title">Output Log</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QPlainTextEdit" name="outputText">
          <property name="font">
           <font>
            <family>Monospace</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="plainText">
           <string notr="true"/>
          </property>
          <property name="textInteractionFlags">
           <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
          </property>
          <property name="placeholderText">
           <string comment="Placeholder text">&lt;Empty output&gt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="processButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string comment="Button text">Process and display</string>
        </property>
        <property name="icon">
         <iconset resource="../resources/resources.qrc">
          <normaloff>:/icons/Run</normaloff>:/icons/Run</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="exportImage">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string comment="Button text">Export</string>
        </property>
        <property name="icon">
         <iconset resource="../resources/resources.qrc">
          <normaloff>:/icons/Export</normaloff>:/icons/Export</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>677</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Fi&amp;le</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Hel&amp;p</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionCMMan"/>
    <addaction name="actionDpicMan"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Setti&amp;ngs</string>
    </property>
    <addaction name="actionPreferences"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSettings"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="previewWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>1</horstretch>
     <verstretch>1</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>130</height>
    </size>
   </property>
   <property name="windowIcon">
    <iconset>
     <normaloff>:/icons/view-preview.png</normaloff>:/icons/view-preview.png</iconset>
   </property>
   <property name="windowTitle">
    <string>&amp;Preview</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>4</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="pycktImageViewer" name="imageViewer">
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Use:&lt;/p&gt;
&lt;ul&gt;
&lt;li&gt; Ctrl + Mouse Wheel to Zoom In/Out&lt;/li&gt;
&lt;li&gt; Click and Move to drag the image inside the preview&lt;/li&gt;
&lt;/ul&gt;
&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QAbstractScrollArea::AdjustToContents</enum>
       </property>
       <property name="dragMode">
        <enum>QGraphicsView::ScrollHandDrag</enum>
       </property>
       <property name="resizeAnchor">
        <enum>QGraphicsView::AnchorUnderMouse</enum>
       </property>
       <property name="viewportUpdateMode">
        <enum>QGraphicsView::FullViewportUpdate</enum>
       </property>
       <property name="rubberBandSelectionMode">
        <enum>Qt::IntersectsItemShape</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/icons/Exit</normaloff>:/icons/Exit</iconset>
   </property>
   <property name="text">
    <string comment="Menu item">&amp;Quit</string>
   </property>
   <property name="statusTip">
    <string comment="Status Bar Message">Close application</string>
   </property>
   <property name="shortcut">
    <string comment="Keyboard Shortcut">Ctrl+Q</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/icons/Open</normaloff>:/icons/Open</iconset>
   </property>
   <property name="text">
    <string comment="Menu item">&amp;Open...</string>
   </property>
   <property name="toolTip">
    <string>Open Drawing</string>
   </property>
   <property name="statusTip">
    <string comment="Status Bar Message">Opens an existing drawing</string>
   </property>
   <property name="shortcut">
    <string comment="Keyboard Shortcut">Ctrl+O</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/icons/About</normaloff>:/icons/About</iconset>
   </property>
   <property name="text">
    <string comment="Menu item">&amp;About PyCirkuit</string>
   </property>
   <property name="statusTip">
    <string comment="Status Bar Message">Application information</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/icons/Settings</normaloff>:/icons/Settings</iconset>
   </property>
   <property name="text">
    <string comment="Menu item">Configure &amp;PyCirkuit...</string>
   </property>
   <property name="statusTip">
    <string comment="Status Bar Message">Opens config dialog</string>
   </property>
   <property name="shortcut">
    <string comment="Keyboard Shortcut">Ctrl+P</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/icons/New</normaloff>:/icons/New</iconset>
   </property>
   <property name="text">
    <string comment="Menu item">&amp;New</string>
   </property>
   <property name="toolTip">
    <string>New Drawing</string>
   </property>
   <property name="statusTip">
    <string comment="Status Bar Message">Creates an empty drawing</string>
   </property>
   <property name="shortcut">
    <string comment="Keyboard Shortcut">Ctrl+N</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/icons/Save</normaloff>:/icons/Save</iconset>
   </property>
   <property name="text">
    <string comment="Menu item">&amp;Save</string>
   </property>
   <property name="statusTip">
    <string comment="Status Bar Message">Saves drawing</string>
   </property>
   <property name="shortcut">
    <string comment="Keyboard Shortcut">Ctrl+S</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/icons/SaveAs</normaloff>:/icons/SaveAs</iconset>
   </property>
   <property name="text">
    <string comment="Menu item">Sa&amp;ve As...</string>
   </property>
   <property name="statusTip">
    <string comment="Status Bar Message">Saves drawing with new name</string>
   </property>
   <property name="shortcut">
    <string comment="Keyboard Shortcut">Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionCMMan">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/icons/CMman</normaloff>:/icons/CMman</iconset>
   </property>
   <property name="text">
    <string>&amp;Circuit Macros manual</string>
   </property>
   <property name="statusTip">
    <string comment="Status Bar Message">Opens Circuit Macros manual</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
  <action name="actionDpicMan">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/icons/CMman</normaloff>:/icons/CMman</iconset>
   </property>
   <property name="text">
    <string>&amp;Dpic manual</string>
   </property>
   <property name="statusTip">
    <string comment="Status Bar Message">Opens Dpic manual</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>pycktTextEditor</class>
   <extends>QTextEdit</extends>
   <header>pycirkuit/texteditor.h</header>
  </customwidget>
  <customwidget>
   <class>pycktImageViewer</class>
   <extends>QGraphicsView</extends>
   <header>pycirkuit/imageviewer.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>sourceText</tabstop>
 </tabstops>
 <resources>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>146</x>
     <y>66</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
